src/compile/utils.ts

```diff
@@ -236,24 +235,10 @@ export function compile(
     finder: ImportFinder,
     remapping: string[]
 ): any {
+    const callbacks = { import: finder };
     const compiler = getCompilerForVersion(version);
 
-    if (satisfies(version, "0.4")) {
-        const input = createCompiler04Input(fileName, content, remapping);
-        const output = compiler.compile(input, 1, finder);
-
-        return output;
-    }
-
-    if (satisfies(version, "0.5")) {
-        const input = createCompiler05Input(fileName, content, remapping);
```

Comment:
Why did we have separate `createCompiler05Input` and `createCompiler04Input` functions in the past?

Reply:
For legacy package versions the `sources` property had following format:
```
```